templates/items/details/details-weapon.hbs

```diff
@@ -73,6 +52,56 @@
     {{/if}}
 </fieldset>
 
+<fieldset>
+    <legend>{{ localize "DND5E.Range" }}</legend>
+
+    {{#if hasRange}}
+
+    <div class="form-group split-group">
+        <label>{{ localize "DND5E.RangeDistance" }}</label>
+        <div class="form-fields">
+
+            {{!-- Normal --}}
+            {{ formField fields.range.fields.value value=source.range.value label="DND5E.Normal" localize=true
+                         hint=false classes="label-top" }}
+
+            {{!-- Long --}}
+            {{ formField fields.range.fields.long value=source.range.long label="DND5E.Long" localize=true
+                         hint=false classes="label-top" }}
+
+            {{!-- Reach --}}
+            {{#if (eq system.attackType "melee")}}
```

Comment:
Right now this appears for all melee weapons, do we want to limit to only melee weapons with the "Reach" property? We could add a `hasReach` getter to `WeaponData` alongside the existing `hasRange` getter to handle that.

Reply:
My main hesitation was around NPC weapons. I wasn't sure if we really wanted to require checking the 'reach' property for something like the Kraken's Tentacle, because that property has well-defined meaning in game terms, which does not apply to most NPC natural weapons.

> This weapon adds 5 feet to your reach when you attack with it, as well as when determining your reach for opportunity attacks with it

On the other hand, our weapon properties don't always map 1:1 with game terms, and we are looser with some of the definitions. So let me know what you think.

Reply:
related conversation from a while back: #1037 
Strixhaven has some npcs with Spell attacks with Reach, e.g.: `Force Strike. Melee or Ranged Spell Attack: +14 to hit, reach 15 ft. or range 120 ft.`